package.json

```diff
@@ -89,10 +91,12 @@
     "yargs": "15.3.1"
   },
   "devDependencies": {
+    "@types/diff": "^5.0.2",
     "@types/find-up": "^2.1.1",
     "@types/fs-extra": "^8.0.1",
     "@types/glob": "^7.1.1",
     "@types/gulp": "^4.0.9",
+    "@types/hogan.js": "^3.0.1",
```

Comment:
Hey @t1m0thyj , 
Thanks for reviewing the changes. 
As for now, we can absolutely remove the generator and directly create the web diff dir as suggested, but in the future, we are planning to build CSS and JS modules locally only so it will even work offline as you are even suggesting ahead. And if that will be the case we will surely need to implement the generator as it is rn. Let me know your view about this? If we want we can remove it for now too. 
And about this dependency actually whenever I remove it the ts compiler throws me an error that missing types of hogan.js that's why I kept it here.

Reply:
Apologies, I had not realized the WebDiffGenerator is a work in progress and I see now that `@types/hogan.js` is required for the build to pass. I think it's fine to leave the WebDiffGenerator as-is, and can approve this PR once other comments have been addressed 🙂

Reply:
No worries, 
Will keep the web diff generator as it is then and I will improve the further required changes soon. :slightly_smiling_face: